<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>749</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>空氣導管X光硬焊瑕疵辨識系統</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="Save_result_path_Button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>儲存檔案路徑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_xml_Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>590</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>儲存參數</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Open_xml_Button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>開啟參數</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="type_of_parts_groupbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="type_of_parts_label">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>種類 type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="type_of_parts_input">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="type_display">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>種類: A</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="width_of_column_groupbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="width_of_column_label">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>寬度 width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="width_of_column_input">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="width_of_column_display">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>寬度: 10</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="area_of_column_groupbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="area_of_column_label">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>面積 area</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="area_of_column_input">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="area_of_column_display">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>面積: 10</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="number_of_section_group">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="number_of_section_label">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>段數 section</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="section7_radiobutton">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="section8_radiobutton">
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微軟正黑體</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="number_of_section_input">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微軟正黑體</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="number_of_section_display">
         <property name="font">
          <font>
           <family>微軟正黑體</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>7段</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Open_file_path_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>971</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>開啟:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_Button">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>590</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>開始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_Button">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>590</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>關閉</string>
   </property>
  </widget>
  <widget class="QListWidget" name="File_list">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>421</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Save_result_path_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>700</y>
     <width>971</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>儲存:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Choose_file_or_path_Button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>590</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>選擇檔案</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="file_or_path_mode_groupbox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="Filemode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>單個檔案</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="Pathmode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>資料夾</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
